.github/workflows/test-saltproc.yml

```diff
@@ -82,15 +82,48 @@ jobs:
           repository: openmc-dev/openmc
           path: openmc
 
-      - name: Build OpenMC
+      - name: Build OpenMC from scratch
         if: steps.dependencies-cache.outputs.cache-hit != 'true'
         run: |
           cd openmc
-          pip install --upgrade-strategy only-if-needed --no-binary=h5py h5py
           python tools/ci/gha-install.py
+          mkdir openmc_src
+          mkdir openmc_src/bin
+          mkdir openmc_src/lib
+          mkdir openmc_src/share
+          mkdir openmc_src/include
+          cp /usr/local/bin/openmc openmc_src/bin/.
+          cp -r /usr/local/lib/x86_64-linux-gnu/ openmc_src/lib/.
+          cp -r /usr/local/share/openmc openmc_src/share/.
```

Comment:
this looks pretty similar to your directions for the cache-dependencies workflow. Are they identical? Maybe we could move the directions to a script that's kept in the worklow directory and then both workflows could source it so we reduce duplication?